### PR TITLE
SCAL-81258-preview-in-playground-url-fixes

### PR DESCRIPTION
**Pull request by pankajp-ts**
Ticket - https://thoughtspot.atlassian.net/browse/SCAL-81258

### CODE REVIEW

File: docs/src/configs/doc-configs.js [hunk 1]
```diff
@@ -3,8 +3,11 @@ module.exports = {
     DOC_NAV_PAGE_ID: 'nav',
     TS_HOST_PARAM: 'tshost',
     TS_ORIGIN_PARAM: 'origin',
+    TS_APP_ROOT_PARAM: 'appRoot',
     TS_PAGE_ID_PARAM: 'pageid',
     NAV_PREFIX: 'navprefix',
     NOT_FOUND_PAGE_ID: '404-error',
     TYPE_DOC_PREFIX: 'typedoc',
+    DEFAULT_HOST: 'https://try-everywhere.thoughtspot.cloud/v2',
+    DEFAULT_APP_ROOT: '/everywhere-standalone',
```
Comment (kailash-choudhary-thoughtspot) [inline docs/src/configs/doc-configs.js:12]: can we change it to /everywhere

Comment (pankajp-ts) [reply docs/src/configs/doc-configs.js]: done

File: docs/src/utils/app-utils.ts [hunk 2]
```diff
@@ -16,22 +20,44 @@ export const queryStringParser = (queryParamStr: string) => {
...
         if (key === TS_ORIGIN_PARAM) {
-            navPrefix += `${TS_ORIGIN_PARAM}=${encodeURIComponent(value)}&`;
+            if (value) {
+                navPrefix += `${TS_ORIGIN_PARAM}=${encodeURIComponent(value)}&`;
+                tsHost = removeTrailingSlash(value.split('#')[0]);
+            }
+        }
+        if (key === TS_APP_ROOT_PARAM) {
+            navPrefix += `${TS_APP_ROOT_PARAM}=${value}&`;
+            appRoot = value;
```
Comment (kailash-choudhary-thoughtspot) [inline docs/src/utils/app-utils.ts:39]: appRoot is not used anywhere

File: docs/src/utils/app-utils.ts [hunk 3]
```diff
@@ -16,22 +20,44 @@ export const queryStringParser = (queryParamStr: string) => {
 
     const entries = new URLSearchParams(queryParamStr).entries();
     let navPrefix = '?';
+    let tsHost = DEFAULT_HOST;
```
Comment (kailash-choudhary-thoughtspot) [inline docs/src/utils/app-utils.ts:23]: can we change the name to tsHostRoute or tsHostUrl this seems confusing

File: docs/src/utils/app-utils.ts (hunk 2 above)
Comment (pankajp-ts) [reply docs/src/utils/app-utils.ts:39]: removed

File: docs/src/utils/app-utils.ts (hunk 3 above)
Comment (pankajp-ts) [reply docs/src/utils/app-utils.ts]: done

**Review by kailash-choudhary-thoughtspot — APPROVED** [review]

**Pull request merged**